dev/NavigationView/NavigationView.idl

```diff
@@ -135,6 +135,12 @@ unsealed runtimeclass NavigationViewTemplateSettings : Windows.UI.Xaml.Dependenc
     static Windows.UI.Xaml.DependencyProperty TopPaneVisibilityProperty { get; };
     static Windows.UI.Xaml.DependencyProperty LeftPaneVisibilityProperty { get; };
     static Windows.UI.Xaml.DependencyProperty SingleSelectionFollowsFocusProperty { get; };
+
+    [WUXC_VERSION_MUXONLY]
+    {
+        String PaneTitle{ get; };
```

Comment:
New APIs require API doc (since this will appear on MSDN). Please do that process before completing this PR. You can model your doc after this one: Microsoft/microsoft-ui-xaml-specs#13.

Reply:
[specs#18](https://github.com/Microsoft/microsoft-ui-xaml-specs/pull/18)